### PR TITLE
Wb/alert toast

### PR DESCRIPTION
**Pull request by WhiteBlock114514**

### CODE REVIEW

File: Features/System/View/ContactInfoView.swift [hunk 1]
```diff
@@ -394,6 +408,7 @@ struct ContactInfoView: View {
     // MARK: Private
 
     @State private var isHakubillDetailShow = false
+    @State private var isWhiteBlockDetailShow = false
```
Comment (CanglongCl) [inline Features/System/View/ContactInfoView.swift:411]: What's this?

File: Features/Manage Accounts/ManageAccountsView.swift [hunk 2]
```diff
@@ -6,9 +6,18 @@
 //
 
 import SwiftUI
+import AlertToast
+
+// MARK: Environment Variable for AlertToast
+
+class AlertToastVariable: ObservableObject {
+    @Published var isDoneButtonTap: Bool = false
```
Comment (CanglongCl) [inline Features/Manage Accounts/ManageAccountsView.swift]: “isDoneButtonTap” should be "isDoneButtonTapped"?

File: Features/Manage Accounts/ManageAccountsView.swift [hunk 3]
```diff
@@ -6,9 +6,18 @@
...
+    @Published var isLoginSuccessed: Bool = false
```
Comment (CanglongCl) [inline Features/Manage Accounts/ManageAccountsView.swift]: "successed" should be "succeeded"

File: Features/System/View/ContactInfoView.swift (hunk 1 above)
Comment (Bill-Haku) [reply Features/System/View/ContactInfoView.swift:411]: his developer contact info

Comment (CanglongCl) [reply Features/System/View/ContactInfoView.swift:411]: But this var is unused

**Review by CanglongCl — APPROVED** [review]

**Pull request merged**